copier/main.py

```diff
@@ -376,13 +376,21 @@ def jinja_env(self) -> SandboxedEnvironment:
         """
         paths = [str(self.template.local_abspath)]
         loader = FileSystemLoader(paths)
+        default_extensions = [
+            "cookiecutter.extensions.JsonifyExtension",
+            "cookiecutter.extensions.RandomStringExtension",
```

Comment:
These 2 extensions are not needed, because `AnsibleCoreFiltersExtension` includes:

- `ans_random`
- `to_json`
- `to_nice_json`